.github/workflows/avro.yaml

```diff
@@ -0,0 +1,104 @@
+name: avro
+
+on:
+  push:
+    branches:
+      - master
+      - main
+    paths:
+      - '**.go'
+      - 'go.*'
+      - Makefile
+  pull_request:
+    paths:
+      - '**.go'
+      - 'go.*'
+      - Makefile
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+env:
+  directory: "."
+  allow_lint_failure: "true"
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up env
+        uses: actions/setup-go@v3
+        with:
+          cache: true
+          check-latest: true
+          cache-dependency-path: ${{ env.directory }}/go.sum
+          go-version-file: ${{ env.directory }}/go.mod
+      - name: Build
+        run: |
+          make build
+  test:
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/heetch/containers/heetch-base-testenv:1
+    services:
+      registry:
+        image: lensesio/fast-data-dev:2.6.2-L0
```

Comment:
Available on aws public mirror?

Reply:
Unfortunately no aws public mirror 😢

Reply:
Not too sure on the throttling bit , we could be but given that there's not much movement going on here we could be okay

Reply:
I believe GitHub hosted runners are logged in to dockerhub.io, so we should be fine

Reply:
😭